certora/specs/ERC20.spec

```diff
@@ -97,7 +87,7 @@ rule transfer(env e) {
     uint256 otherBalanceBefore     = balanceOf(other);
 
     // Safe require as it's proven by the rule twoBalancesCannotExceedTotalSupply.
-    require balanceOf(holder) + balanceOf(recipient) <= totalSupply();
+    if (holder != recipient) { require balanceOf(holder) + balanceOf(recipient) <= totalSupply(); }
```

Comment:
Changed in https://github.com/morpho-org/morpho-token-upgradeable/pull/102/commits/b888a1ffd1da1eca4f383c4b6c4fd1897e4ad3b4.